apps/auth/.env.production

```
@@ -1,9 +1,24 @@
-NEXTAUTH_URL=https://auth.klicker.uzh.ch
 EDUID_CLIENT_ID=uzh_klicker_auth
 EDUID_WELL_KNOWN=https://login.eduid.ch/.well-known/openid-configuration
 NEXT_PUBLIC_EDUID_ID=eduid
-NEXT_PUBLIC_DEFAULT_REDIRECT=https://manage.klicker.uzh.ch
-NEXT_PUBLIC_AUTH_URL=https://auth.klicker.uzh.ch
-NEXT_PUBLIC_MANAGE_URL="https://manage.klicker.uzh.ch"
-NEXT_PUBLIC_ASSESSMENT_URL="https://assessment.klicker.uzh.ch"
+
+APP_ORIGIN_API=https://backend-sls.klicker.uzh.ch
+APP_ORIGIN_PWA=https://pwa.klicker.uzh.ch
+APP_ORIGIN_MANAGE=https://manage.klicker.uzh.ch
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_ORIGIN_MANAGE key should go before the APP_ORIGIN_PWA key

(UnorderedKey)